cadl-tests/src/main/java/com/cadl/builtin/BuiltinClient.java

```diff
@@ -41,6 +41,13 @@ public final class BuiltinClient {
      *
      * <pre>{@code
      * {
+     *     formatString (Required): {
+     *         base64Encoded: byte[] (Required)
+     *         binary: byte[] (Required)
+     *         dateTime: OffsetDateTime (Required)
+     *         dateTimeRfc1123: OffsetDateTime (Required)
```

Comment:
@haolingdong-msft 

Would you add a bug for yourself?

type of `base64Encoded` and `dateTimeRfc1123` seems not correct.

Reply:
issue created: https://github.com/Azure/autorest.java/issues/1666